os/kernel/Kconfig

```diff
@@ -1080,6 +1080,17 @@ config SCHED_LPWORKSTACKSIZE
 endif # SCHED_LPWORK
 endmenu # Work Queue Support
 
+menu "fault Manager"
+menuconfig FAULT_MGR
+	bool "Enable Fault Manager"
+	default n
+	depends on APP_BINARY_SEPARATION
+	---help---
+		Fault Manager interfaces.  These interface are used
+		to manage any fault occurs in a system. It will either restert app
+		or reboor the system according to the configuration.
```

Comment:
Please modify the typo. reboor -> reboot

Reply:
Done